example/app.js

```diff
@@ -113,10 +120,8 @@ class App extends React.Component {
   /**
    * Handle function for form data, triggered whenever the user modifies form values.
    */
-  onUpdate(evt, name, field, value) {
-    let values = this.state.values;
-    values[name] = value;
-    this.setState({ values });
+  onUpdate(evt, name, field, value, updatedValues) {
+    this.setState({ values: Object.assign(this.state.values, updatedValues)});
```

Comment:
This should not be necessary, the sum up `onUpdate` calls should already cover the full state of all children.